src/plugin/plugin-manager.js

```diff
@@ -30,6 +30,12 @@ export default class PluginManager {
    * @private
    */
   _plugins: Map<string, BasePlugin> = new Map();
+  /**
+   * Is disabled plugin map.
+   * @type {Map}
+   * @private
+   */
+  _isDisabledPluginMap: Map<string, boolean> = new Map();
```

Comment:
`disabledPlugins` Is better fitting and u can save the bytes and set it as an array as the array only needs to have disabled plugins in it so you can use `disabledPlugins.contans(pluginNmae)`

Reply:
Consider that this implementation is a bitconfusing cause if a plugin was not set with disable as Boolean  it will not be in the map at all and this is a bit misleading.
If it is not in it what does it mean? If it is and has true what does it mean. Leads to guessing.
As I already mentioned in the closed PR in Kaltura player we will add an API to enable/disable and get state of of plugins from plugins manager, so this is a good time to think about it.
If you don’t have a better idea at the moment at least make sure to add a plugin if it was init to the map with value false so we have consistency.